runtime/interpreter/encoding_test.go

```diff
@@ -126,6 +127,24 @@ func testEncodeDecode(t *testing.T, test encodeDecodeTest) {
 	}
 }
 
+// encodeStorable wraps storable.Encode()
+func encodeStorable(storable atree.Storable, encMode cbor.EncMode) ([]byte, error) {
+	var buf bytes.Buffer
```

Comment:
Would it make sense to pool these buffers, like in atree?

Reply:
> Would it make sense to pool these buffers, like in atree?

It is probably not necessary here because this function is used only in tests.

Reply:
Ah, right, I missed that this was in tests 👍